docs/reference/commandline/login.md

```diff
@@ -162,7 +164,7 @@ designated programs to handle credentials for *specific registries*. The default
 credential store (`credsStore` or the config file itself) will not be used for
 operations concerning credentials of the specified registries.
 
-### Logging out
+#### Usage
```

Comment:
Thinking if we should use a more descriptive title; perhaps "Credential helper configuration", or "Configure credential helpers" ?

Reply:
done